mesa/mesaPROTON_OC.py

```diff
@@ -1161,27 +1168,9 @@ def let_migrants_in(self):
                 new_agent.wealth_level = job.job_level
                 new_agent.migrant = True
 
-
-# 778 / 1700
-# next: testing an intervention that removes kids and then returning them.   
-# test OC members formation
-
-# next code, needed to run: social proximity
-
-# next: repair the tests, stop pushing forward. Repair those tests!
-
-# end class. From here, static methods
-
-# warning: for now we don't load up the partner in the partner network
-def conclude_wedding(ego, partner):
-    for x in [ego, partner]:
-        for y in x.neighbors["household"]:
-            y.neighbors["household"].discard(x)  # should be remove(x) once we finish tests
-    ego.neighbors["household"].add(partner)
-    partner.neighbors["household"].add(ego)
-    ego.neighbors["partner"].add(partner)
-    partner.neighbors["partner"].add(ego)
-staticmethod(conclude_wedding)
+    def cal_criminal_tendency_addme(self):
+        min_criminal_tendencies = np.min([agent.criminal_tendency for agent in self.schedule.agents])
```

Comment:
We use np.min here and only min in the weighted_n_of - why? We might just get rid of this one now that the above is negative-safe.

Reply:
Done!